src/main/java/com/exasol/releaserobot/release/GitHubReleaseMaker.java

```diff
@@ -49,7 +48,7 @@ public boolean makeRelease() {
             this.gitHubPlatform.makeNewGitHubRelease(release);
             this.releaseReport.addSuccessfulRelease(this.gitHubPlatform.getPlatformName());
             return true;
-        } catch (final RuntimeException runtimeException) {
+        } catch (final GitHubException runtimeException) {
```

Comment:
here for example, you are expect to recover from this error (as you are doing it now), so it's good to use a checked exception here.
You might consider at some point create a more domain specific exception for this, like "ReleaseException" or similar.

Might also be in a following PR